integration_tests/src/main/python/orc_write_test.py

```diff
@@ -91,6 +91,20 @@ def test_write_round_trip(spark_tmp_path, orc_gens, orc_impl):
             data_path,
             conf={'spark.sql.orc.impl': orc_impl, 'spark.rapids.sql.format.orc.write.enabled': True})
 
+@pytest.mark.parametrize('orc_gens', orc_write_gens_list, ids=idfn)
+@pytest.mark.parametrize('orc_impl', ["native", "hive"])
+@allow_non_gpu(*non_utc_allow)
+def test_write_more_than_one_stripe_round_trip(spark_tmp_path, orc_gens, orc_impl):
+    gen_list = [('_c' + str(i), gen) for i, gen in enumerate(orc_gens)]
+    data_path = spark_tmp_path + '/ORC_DATA'
+    assert_gpu_and_cpu_writes_are_equal_collect(
+            # Generate a large enough dataframe to produce more than one stripe(typically 64MB)
+            # Preferably use only one partition to avoid splitting the data
+            lambda spark, path: gen_df(spark, gen_list, 12800, num_slices=1).write.orc(path),
```

Comment:
Question: Where does the 12800 number come from? Do we know it will be greater than 64m (orc stripe size) for all the datagens you tested?

Reply:
This number comes from my experiment.(

Reply:
In general CUDF will split the data by rows and by size.

https://github.com/rapidsai/cudf/blob/f54c1a5ad34133605d3b5b447d9717ce7eb6dba0/cpp/include/cudf/io/orc.hpp#L585-L587

https://github.com/rapidsai/cudf/blob/f54c1a5ad34133605d3b5b447d9717ce7eb6dba0/cpp/include/cudf/io/orc.hpp#L41-L42

In parquet the split rows in 20,000 but for ORC it is 1,000,000. I am not sure how 12,800 booelan values produces a more than one stripe.  I would really like to understand this better because I would expect that to be no where close to the row group count we expect to cause multiple slices.